### PR TITLE
Bud 110 improve create activity

### PR DESCRIPTION
**Pull request by JadaTBK**
General improvements to the create activity screen.
- faster search
- save location_text in firestore for displaying activity

### CODE REVIEW

File: Buddies/Keys.plist.enc [hunk 1]
```diff
@@ -1,2 +0,0 @@
-�N{�$��wA�Q���yp�!����_����w<��@c!�W)��`
```
Comment (noahtallen) [inline Buddies/Keys.plist.enc]: Is this supposed to be deleted?

(It's needed for the Travis build)

File: Buddies/CreateActivity/CreateActivityVC.swift [hunk 2]
```diff
@@ -337,6 +343,7 @@ extension CreateActivityVC: MKLocalSearchCompleterDelegate {
     
     private func completerDidUpdateResults(completer: MKLocalSearchCompleter) {
         searchResults = completer.results
+        
     }
```
Comment (noahtallen) [inline Buddies/CreateActivity/CreateActivityVC.swift:348]: Do we save the nice text to firebase yet?

**Review by noahtallen — COMMENTED** [review]
I'll test it out in a bit ;)

**Review by noahtallen — APPROVED** [review]
Looks good!

One thing, if you type text into location but you don't actually select an entry and juts leave the text, it just says that you haven't entered details. we should probably make that more clear in the future

**Comment by lukemeierj** [comment]
Following up on Noah's point:

If you don't fill out a field,  could the pop-up say which field needs to be filled in? I don't think it does this rn

File: Buddies/CreateActivity/CreateActivityVC.swift (hunk 2 above)
Comment (JadaTBK) [reply Buddies/CreateActivity/CreateActivityVC.swift:348]: ye we do

**Pull request merged**